conf/config-default.yaml

```diff
@@ -338,6 +338,7 @@ plugins:                          # plugin list (sorted by priority)
   - traffic-split                  # priority: 966
   - redirect                       # priority: 900
   - response-rewrite               # priority: 899
+  - openwhisk                      # priority: 601
```

Comment:
Please use the similar priority like azure-functions

Reply:
Should I use the same `-1900` priority or add one `-1901`? Because they have similar functions and should not be enabled at the same time.

Reply:
`-1901`.